clients/src/main/java/org/apache/kafka/common/requests/AlterShareGroupOffsetsResponse.java

```diff
@@ -63,4 +66,46 @@ public static AlterShareGroupOffsetsResponse parse(Readable readable, short vers
             new AlterShareGroupOffsetsResponseData(readable, version)
         );
     }
+
+    public static class Builder {
+        AlterShareGroupOffsetsResponseData data = new AlterShareGroupOffsetsResponseData();
+        HashMap<String, AlterShareGroupOffsetsResponseTopic> topics = new HashMap<>();
```

Comment:
Have you considered making the `AlterShareGroupOffsetsResponseData` use topic name as key in protocol file?

Reply:
Actually, I think that's a potential improvement in a few places in the KIP-932 requests and responses.